infra/build/functions/base_images.py

```diff
@@ -78,15 +78,18 @@ def _get_introspector_base_images_steps(images, tag_prefix=TAG_PREFIX):
       'name': 'gcr.io/cloud-builders/git',
   }]
```

Comment:
nit: instead of creating this list and appending to it twice, put all 3 steps in the list from the beginning

Reply:
Done.